spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -487,6 +487,7 @@
 "https://creativecommons.org/licenses/by/4.0": CC-BY-4.0
 "https://creativecommons.org/publicdomain/zero/1.0/": CC0-1.0
 "https://dotnet.microsoft.com/en/dotnet_library_license.htm": LicenseRef-scancode-ms-net-library-2018-11
+"https://github.com/dotnet/core-setup/blob/master/LICENSE.TXT": MIT
```

Comment:
For the record: The content under this link might be changing so the entry is risky, but it was recently decided to continue taking this kind of risk and clean it up later. Similar comment applies to some of the other entries of this PR.

Reply:
Right, see also what I wrote above in the initial description of this PR.